app/src/main/java/com/github/fribourgsdp/radio/data/view/SongSwipeHelper.kt

```diff
@@ -0,0 +1,77 @@
+package com.github.fribourgsdp.radio.data.view
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.view.MotionEvent
+import android.view.View
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.ItemTouchHelper.LEFT
+import com.github.fribourgsdp.radio.R
+import com.github.fribourgsdp.radio.data.Song
+import com.google.android.material.snackbar.Snackbar
+
+class SongSwipeHelper (val recyclerView: RecyclerView, private val songList : MutableList<Song>,
+                       val rootView: View, val context: Context) : ItemTouchHelper.Callback() {
+    override fun getMovementFlags(
+        recyclerView: RecyclerView,
+        viewHolder: RecyclerView.ViewHolder
+    ): Int {
+        return makeMovementFlags(0, LEFT)
+    }
+
+    override fun onMove(
+        recyclerView: RecyclerView,
+        viewHolder: RecyclerView.ViewHolder,
+        target: RecyclerView.ViewHolder
+    ): Boolean {
+        return false
```

Comment:
Why would `onMove` be always false?

Reply:
I assume move is for rearranging cards in the recyclerview. This is not something we are trying to do.